### PR TITLE
chore: drop legacy buttons

### PR DESCRIPTION
**Pull request by waterplea**
Closes #

### CODE REVIEW

**Review by splincode — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/addon-charts/components/legend-item/legend-item.module.ts
Comment (nsbarsukov) [inline projects/addon-charts/components/legend-item/legend-item.module.ts:1]: Review ...

Block merge

File: projects/demo/src/modules/components/button/index.html [hunk 1]
```diff
@@ -1,61 +1,18 @@
...
-        </tui-doc-example>
-
-        <tui-doc-example
-            id="icon"
-            heading="Icons"
-            [content]="example3"
-        >
-            <tui-button-example-3 />
-        </tui-doc-example>
+        New version for Button component following updated design specs.
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/button/index.html:7]: Should we drop this line for `4.x` ?

File: projects/demo/src/modules/components/data-list/examples/1/index.html [hunk 2]
```diff
@@ -1,8 +1,9 @@
+<!--TODO: TUI_ARROW migration-->
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/data-list/examples/1/index.html:1]: Mention this TODO in this issue https://github.com/taiga-family/taiga-ui/issues/5808

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/components/data-list/examples/1/index.html (hunk 2 above)
Comment (waterplea) [reply projects/demo/src/modules/components/data-list/examples/1/index.html:1]: I will do it after I implement what to migrate to :)

**Pull request merged**